frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/emoji_picker/emoji_menu_item.dart

```diff
@@ -15,15 +15,15 @@ SelectionMenuItem emojiMenuItem = SelectionMenuItem(
   keywords: ['emoji'],
   handler: (editorState, menuService, context) {
     final container = Overlay.of(context);
-    showEmojiPickerMenu(
+    _showEmojiPickerMenu(
```

Comment:
use showEmojiPickerMenu instead.

Reply:
I wanted to reuse the showEmojiPickerMenu function but it requires a SelectionMenuService object as input. Instead, I thought it would be better to wrap the _showEmojiPickerMenu function which only requires the height and offset so that it can be reused for the emojiShortcutEvent.

Reply:
What I mean is using

```
showEmojiPickerMenu(
      container, 
      editorState, 
      alignment, 
      menuService.offset,
) 
```

instead of 

```
_showEmojiPickerMenu(
      container,
      editorState,
      menuService,
)
```

Reply:
I've just submitted a commit removing `_showEmojiPickerMenu` following this suggestion, the feature continues to work the same.